### PR TITLE
Feat/add charactercard component

### PR DESCRIPTION
**Pull request by karlknowit**
## Explain the changes you’ve made
Added a new component called CharacterCard which is going to be used for displaying and selecting people to book meeting with in the application.
Also added jest unittests and updated it's dependencies and test-utils. 
Added correct typings for styled-components for react-native

## Explain why these changes are made
These changes are preparations in order to continue with the booking service in the application. Instead of making a PR with all new components needed, we are separate them into smaller chunks which could be merged into dev-branch directly and independently. 

## Explain your solution
My solution consists of react-native components put together into one single card component.

## How to test the changes?

Concrete example:
1. Checkout this branch
2. Swap the application to run storybook
3. Fire upp the simulator by running the command`yarn ios`

## Was this feature tested in the following environments?
- [x] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots (optional)
image
image

### CODE REVIEW

File: source/components/molecules/CharacterCard/CharacterCard.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  jobTitle: string;
+  selected: boolean;
+  onCardClick: () => void;
+};
+const CharacterCard = (props: CharacterCardType): JSX.Element => {
+  const {
+    icon,
+    selected = false,
+    title,
+    appartment,
```
Comment (sepehrnoor) [inline source/components/molecules/CharacterCard/CharacterCard.tsx:21]: Should be 'department'

**Review by sepehrnoor — APPROVED** [review]
Looks good. There's just a tiny typo in a prop name.

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1f8zu41 Skapa kort för att visa handläggare](https://app.clickup.com/t/1f8zu41)

File: source/components/molecules/CharacterCard/CharacterCard.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from "react";
+import { ThemeContext } from "styled-components/native";
+import Checkbox from "../../atoms/Checkbox";
+import { colorPalette } from "../../../styles/palette";
+
+import { Card, Section, Image, CardTitle, CardSubtitle } from "./styled";
+
+type CharacterCardType = {
+  icon: any;
```
Comment (KenthLjungHBG) [inline source/components/molecules/CharacterCard/CharacterCard.tsx:9]: We should always avoid using `any` unless really necessary. Can `icon` be narrowed down?

File: source/components/molecules/CharacterCard/CharacterCard.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+const CharacterCard = (props: CharacterCardType): JSX.Element => {
+  const {
+    icon,
+    selected = false,
+    title,
+    department,
+    jobTitle,
+    onCardClick,
+  } = props;
+  const { colors } = useContext<any>(ThemeContext);
```
Comment (KenthLjungHBG) [inline source/components/molecules/CharacterCard/CharacterCard.tsx:25]: Same with `any` here - there should be a specific type for `ThemeContext` right?

**Review by KenthLjungHBG — CHANGES_REQUESTED** [review]
Looks good. Just had a few small pointers 🙂.

File: source/components/molecules/CharacterCard/CharacterCard.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import React, { useContext } from "react";
+import { ThemeContext } from "styled-components/native";
+import Checkbox from "../../atoms/Checkbox";
+import { colorPalette } from "../../../styles/palette";
+
+import { Card, Section, Image, CardTitle, CardSubtitle } from "./styled";
+
+type CharacterCardType = {
```
[the remark this answers is not in the document]
Comment (KenthLjungHBG) [reply source/components/molecules/CharacterCard/CharacterCard.tsx:8]: > `type` and `interface` are pretty much the same in Typescript and you can use both. Im all in using interface as well.

There are some minor differences, but coming from a background in C++ i prefer to have object structures as `interface` and basic types, aliases, enums etc. as `type`. This is only my personal preferences though, nothing we've discussed in the team yet 🙂.

File: source/components/molecules/CharacterCard/CharacterCard.tsx (hunk 2 above)
Comment (karlknowit) [reply source/components/molecules/CharacterCard/CharacterCard.tsx]: Found type for icon.Will use it.

**Review by sepehrnoor — APPROVED** [review]

**Review by KenthLjungHBG — APPROVED** [review]

**Pull request merged**